test/xUnit/csharp/test_SecuritySupport.cs

```diff
@@ -12,19 +12,14 @@ public static class SecuritySupportTests
         [Fact]
         public static void TestScanContent()
         {
-            Assert.Equal(AmsiUtils.AmsiNativeMethods.AMSI_RESULT.AMSI_RESULT_NOT_DETECTED, AmsiUtils.ScanContent(string.Empty, string.Empty));
-        }
-
-        [Fact]
-        public static void TestCloseSession()
-        {
-            AmsiUtils.CloseSession();
-        }
-
-        [Fact]
-        public static void TestUninitialize()
-        {
-            AmsiUtils.Uninitialize();
+            try
+            {
+                Assert.Equal(AmsiUtils.AmsiNativeMethods.AMSI_RESULT.AMSI_RESULT_NOT_DETECTED, AmsiUtils.ScanContent(string.Empty, string.Empty));
+            }
+            finally
+            {
+                AmsiUtils.Uninitialize();
```

Comment:
Note: `Uninitialize` calls `CloseSession` internally.

Reply:
I don't think the explanation is needed here. It's more confusing as you won't see the original test cases anymore.